__tests__/integration/verifyTemplateTitle.test.js

```diff
@@ -3,7 +3,7 @@
 import pupExpect from 'expect-puppeteer'
 import { testUserLogin } from './loginHelper'
 
-describe('Verify the resource template title is displayed in the editor', () => {
+describe('Verify the resource template label is displayed in the editor', () => {
```

Comment:
so it's a label in the context of the resource template.  It's a title in the context of the html.  You may prefer title here to label.